### PR TITLE
Draft Release 5 changelog

### PR DESCRIPTION
**Pull request by StachuDotNet**
Covers work released in August of 2022.

Seems like a very productive month to me!

### CODE REVIEW

**Comment by StachuDotNet** [comment]
Noticed an issue (beyond the current build failure):
The Tuple2 and Tuple3 functions are currently disabled, per https://github.com/darklang/dark/pull/4342, regardless of the flag being set in settings.

File: docs/reference/languagedetails.md [hunk 1]
```diff
@@ -101,6 +101,56 @@ y
...
+- Dark's Standard Library includes `Tuple2` and `Tuple3` modules, which provide
+  functions used to work with tuples of size 2 and 3.
+- Pattern matching with `match` supports tuples:
+
+  ```fsharp
+  let headers = Dict::toList request.headers
+  let contentTypePlain =
+    List::findFirst headers \header ->
+      let (key, value) = Tuple2::mapFirst (\key -> String::toLower key) header
+      match key, value
```
Comment (pbiggar) [inline docs/reference/languagedetails.md:133]: Should these have parentheses?

File: docs/reference/languagedetails.md [hunk 2]
```diff
@@ -101,6 +101,56 @@ y
...
+The following constructs are planned but not yet implemented:
+
+- Tuples can be deconstructable into their parts in a `let` expression:
+
+  ```fsharp
+  let philadelphia = (39.9526, 75.1652)
+  let (lat, long) = philadelphia
+  ```
+
+- User functions may accept and return Tuples (TODO: I'm not sure how to phrase
```
Comment (pbiggar) [inline docs/reference/languagedetails.md:151]: I thought tuple types could not be specified in a function signature.

**Review by pbiggar — APPROVED** [review]

**Comment by pbiggar** [comment]
There's a few changes needed, but I'd like to merge quickly, so they can be done after instead.

File: docs/reference/languagedetails.md (hunk 2 above)
Comment (StachuDotNet) [reply docs/reference/languagedetails.md:151]: this is in the 'The following constructs are planned but not yet implemented:' section

File: docs/reference/languagedetails.md (hunk 1 above)
Comment (StachuDotNet) [inline docs/reference/languagedetails.md:133]:
```suggestion
      match (key, value)
```
yes!

**Comment by StachuDotNet** [comment]
fixing the build issue then I'll merge

**Comment by StachuDotNet** [comment]
I'm not able to merge PRs in this repo.

**Comment by pbiggar** [comment]
> I'm not able to merge PRs in this repo.

Moved you to "maintain" permissions

**Pull request merged**